### PR TITLE
Workaround: Remove -R option

### LINKED ISSUE #67: Conflict single letter option "-R"
pytest-remotedata defines single letter option "-R". Unfortunately pytest-leaks plugin also define it.
It is better to avoid define single letter option because of conflict.

Here is an example.
https://github.com/miurahr/aqtinstall/pull/634#issuecomment-1374884570

Could you remove a "-R" single letter option?

### PR DESCRIPTION
**Pull request by bsipocz**
This is to fix #67, and a follow-up PR to pytest-astropy is coming.

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -1,7 +1,8 @@
 0.4.1 (unreleased)
 ==================
 
-- No changes yet.
+- Reverting the short option of ``-R`` due to a clash with ``pytest-leaks``.
+  The short option is added to ``pytest-astropy`` instead. [#70]
```
Comment (pllim) [inline CHANGES.rst:5]: But won't that make any package that install `pytest-astropy` to be incompatible with `pytest-leaks`?

Comment (bsipocz) [reply CHANGES.rst:5]: Yes, it would. But -R was around in the astropy ecosystem forever, so I don't expect much clash in real life scenarios.

Comment (pllim) [reply CHANGES.rst:5]: I guess it is a nice compromise. Packages don't really need the meta package to function if there is indeed a clash down the road.

Comment (pllim) [reply CHANGES.rst:5]: What is the order of operation? Does https://github.com/astropy/pytest-astropy/pull/55 go in first or this one?

**Review by pllim — APPROVED** [review]
LGTM but I'll let you manage the merge order as you desire. Thanks!

**Pull request merged**